core/src/main/java/org/apache/iceberg/CachingCatalog.java

```diff
@@ -110,6 +110,8 @@ private Cache<TableIdentifier, Table> createTableCache(Ticker ticker) {
           .removalListener(new MetadataTableInvalidatingRemovalListener())
           .executor(Runnable::run) // Makes the callbacks to removal listener synchronous
           .expireAfterAccess(Duration.ofMillis(expirationIntervalMillis))
+          .expireAfterWrite(
```

Comment:
I don't think we need/should specify `expireAfterWrite` in addition to `expireAfterAccess`:

* The javadoc of `expireAfterWrite` mentions: `Specifies that each entry should be automatically removed from the cache once a fixed duration has elapsed after the entry's creation, or the most recent replacement of its value.`

* The javadoc of `expireAfterAccess` mentions: `Specifies that each entry should be automatically removed from the cache once a fixed duration has elapsed after the entry's creation, the most recent replacement of its value, or its last access.`

Reply:
In fact, my local code does not have `expireAfterAccess`. I kept `expireAfterAccess` to see how reviewer thinks about this problem. I set the `expireAfterWrite` time to be longer than `expireAfterAccess` because I want to use it as a cache ttl.